operations/node_management/Adding_a_Liquid-cooled_blade_to_a_System.md

```diff
@@ -345,13 +345,45 @@ This procedure will add a liquid-cooled blades from a HPE Cray EX system.
 
 #### Power on and boot the nodes
 
-16. Use boot orchestration to power on and boot the nodes. Specify the appropriate BOS template for the node type.
+Use boot orchestration to power on and boot the nodes. Specify the appropriate BOS template for the node type.
 
-    ```bash
-    ncn-m001# BOS_TEMPLATE=cos-2.0.30-slurm-healthy-compute
-    ncn-m001# cray bos session create --template-uuid $BOS_TEMPLATE \
-      --operation reboot --limit x1005c3s0b0n0,x1005c3s0b0n1,x1005c3s0b1n0,x1005c3s0b1n1
-    ```
+16. Determine how the BOS Session template references compute hosts.
+    Typically, they are referenced by their "Compute" role. However, if they are referenced by xname, then these new nodes should added to the BOS Session template.
+      ```bash
+      ncn-m001# BOS_TEMPLATE=cos-2.0.30-slurm-healthy-compute
+      ncn-m001# cray bos sessiontemplate describe $BOS_TEMPLATE --format json|jq '.boot_sets[] | select(.node_list)'
+      ```
+    If this query returns empty, then skip to sub-step 2.
+    If this query returns with data, then one or more boot sets within the BOS Session template reference nodes explicity by xname. Consider adding your new nodes to this list (sub-step 1) or adding them on the command line (sub-step 2).
+
+      i. Adding new nodes to your list.
```

Comment:
These sub-steps should all be numbered normally in the markdown (1. , 2. , 3., etc.). Using roman numerals or lettered steps doesn't render correctly.

Here is what it currently looks like:
![image](https://user-images.githubusercontent.com/78099781/155204544-063f5c82-a8cc-4e14-84e1-2edf8714c844.png)